lock-impl/src/main/java/com/palantir/lock/impl/LockServiceImpl.java

```diff
@@ -999,9 +994,6 @@ public LockServerOptions getLockServerOptions() {
             @Override public TimeDuration getMaxAllowedClockDrift() {
                 return maxAllowedClockDrift;
             }
-            @Override public TimeDuration getMaxAllowedBlockingDuration() {
```

Comment:
Mention that this option is now replaced by a config parameter in the atlas config.